README.md

```diff
@@ -33,7 +33,7 @@ yarn add vk-io
 
 ### NPM
 ```shell
-npm install vk-io --save
+npm i vk-io
```

Comment:
Why? I think it makes total sense to use it, since it does the same thing but 13 characters shorter, plus the [npm package page](https://www.npmjs.com/package/vk-io) uses it.